src/Microsoft.OData.Client/SaveResult.cs

```diff
@@ -344,11 +344,6 @@ protected override DataServiceResponse HandleResponse()
                 ex = e;
             }
 
-            if (ex != null)
-            {
-                throw new DataServiceRequestException(Strings.DataServiceException_GeneralError, ex, service);
-            }

```

Comment:
1) so, why do you still keep the variable "ex" ?
2) Why do you still keep the try{}catch{} ?